### DISCUSSION TITLE
Local offset calibration uses average or median?

### DISCUSSION THREAD

**Discussion by Alsen99** [ppy/osu#20243] [Q&A] [answered]
I would like to ask a question so please don't take this as a complaint or anything since if this doesn't change anything, I would just adjust the offset manually, Does local offset calibration uses average or median?

I mean if we assume the hit distribution is normal then the average and median should be the same so it doesn't matter

But what if the distribution is say weibull? Shouldn't the calibration is based off median instead of average? Since median is where we hit the most and average is the average of all hits.

**Comment by peppy** [comment, marked as answer]
average currently. see https://github.com/ppy/osu/discussions/19763

**Reply by Alsen99** [reply]
Oh! Thank you for the answer!
